features/testbot/distance_matrix.feature

```diff
@@ -715,11 +716,14 @@ Feature: Basic Distance Matrix
             | fghij  |
 
         When I route I should get
-            | from | to | route              | distance |
-            | 1    | 2  | abcdef,fghij,fghij | 999.9m  |
+            | from | to | route              | distance   |
+            | 1    | 2  | abcdef,fghij,fghij | 1000.1m    |
+            | 1    | 3  | abcdef,fghij,fghij | 1400.1m    |
+            | 2    | 3  | fghij,fghij        | 400m       |
+            
 
-        # TODO: this is "correct", but inconsistent with viaroute
```

Comment:
This test now ensures that we fixed https://github.com/Project-OSRM/osrm-backend/issues/5151 (also checked that manually via requests similar to what was provided in the issue)